fMRISurface/GenericfMRISurfaceProcessingPipeline.sh

```diff
@@ -72,6 +72,10 @@ log_Msg "GrayordinatesResolution: ${GrayordinatesResolution}"
 log_Msg "RegName: ${RegName}"
 log_Msg "RUN: ${RUN}"
 
+if [ "${RegName}" = "FS" ] ; then
+  logWarn "WARNING: FreeSurfer registration is deprecated in the HCP Pipelines as it results in poorer cross-subject functional and cortical areal alignment relative to MSMSulc. Additionally, FreeSurfer registration results in dramatically higher surface distortion (both isotropic and anisotropic). These things occur because FreeSurfer's registration has too little regularizattion of folding patterns that are imperfectly correlated with function and cortical areas, resulting in overfitting of folding patterns. See Robinson et al 2014, 2018 Neuroimage, and Coalson et al 2018 PNAS for more details."
```

Comment:
It's `log_Warn`.  Typo in "regularizattion".   I would start it with "WARNING: FreeSurfer's surface registration (based on cortical folding) is deprecated ...".   Possibly split into individual lines, pending @coalsont's concurrence.

Reply:
I fixed the bug and changed the wording. Wating on whether to break into multiple lines.

Reply:
Split into multiple lines.